src/main/groovy/org/mockito/release/internal/gradle/ReleaseNotesPlugin.java

```diff
@@ -158,4 +162,16 @@ private static void configureNotableReleaseNotes(Project project) {
             fetcherTask.getNotesGeneration().setHeadVersion(project.getVersion().toString());
         }
     }
+
+    private static void addToCommittedChanges(final Project project, final Task task, final File changedFile, final String message){
+        project.getPlugins().withType(GitPushPlugin.class, new Action<GitPushPlugin>() {
+            @Override
+            public void execute(GitPushPlugin gitPushPlugin) {
+                final GitCommitTask gitCommitTask = (GitCommitTask) project.getTasks().getByName(GitPushPlugin.GIT_COMMIT_TASK);
+                gitCommitTask.mustRunAfter(task);
```

Comment:
We already have 'dependsOn' inside 'addChange' method so this line is probably not needed.

I need to think a little bit about when to use 'dependsOn' and when to use 'mustRunAfter' but it should not block you in a any way. 'mustRunAfter' is convenient because allows to us run single task easily, in case we just want to run git commit task. However, it introduces complexity because we need to manage tasks dependsOn somewhere :)